src/main.py

```diff
@@ -1,50 +1,61 @@
 #!/usr/bin/env python
-from multiprocessing import Process
 import logging
+import sys
+from multiprocessing import Process
 
 from extract.main import Extract
 from transform.main import Transform
 
 
+def extract_and_load(address:str) -> None:
+    extract = Extract(address)
+    extract()
+
+def transform_and_load(to_transform:str) -> None:
+    transform = Transform(to_transform)
+    transform()
+
 def main():
     """Starts the whole ETL pipeline. Creates two separate processes.
     One for extraction, and one for transforming.
     """
 
     logging.basicConfig(
-        filename="example_rumble_kong_league.log",
```

Comment:
I would like to maintain the details for other examples like rumble kong league. Let's think of a good way to have all that information here. I have added RKL club nft auctions in the original repo, so would like to have all of that info here in one place. And be able to just change one value and have it all use the correct details elsewhere in here.